sysboundary/sysboundary.cpp

```diff
@@ -636,8 +636,11 @@ bool SysBoundary::classifyCells(dccrg::Dccrg<spatial_cell::SpatialCell, dccrg::C
 
                      technicalGrid.get(x, y, z)->sysBoundaryLayer = layer;
 
-                     if (layer > 2 && technicalGrid.get(x, y, z)->sysBoundaryFlag != sysboundarytype::NOT_SYSBOUNDARY) {
+                     if (layer > 2 && (technicalGrid.get(x,y,z)->sysBoundaryFlag == sysboundarytype::IONOSPHERE || 
+                                       technicalGrid.get(x,y,z)->sysBoundaryFlag == sysboundarytype::CONDUCTINGSPHERE)) {
                         technicalGrid.get(x, y, z)->sysBoundaryFlag = sysboundarytype::DO_NOT_COMPUTE;
+                     } else if (layer > 2 && technicalGrid.get(x, y, z)->sysBoundaryFlag != sysboundarytype::NOT_SYSBOUNDARY) {
+                        technicalGrid.get(x, y, z)->sysBoundaryFlag = sysboundarytype::BOUNDARY_PADDING;
                      }
                   }
                }
```

Comment:
I recommend adding a comment to line 647/650 that the ghost update is required becaues belongsToLayer() requires up-to-date ghost information.

Also a comment to line 660/663 that because the layers count is increased so much for FSgrid cells, outer boundary cells should all be handled by the outer loop. Do they? Or we could add the PADDING setting there as well just to be sure.

Reply:
I had for a bit wondered why this only handles ionosphere/copysphere, and not the outer wall boundary types, but the reason is that any DNC/BP cells at the outer walls are already handled by the previous loop. And that is exactly because the value `MAX_NUMBER_OF_BOUNDARY_LAYERS` is so large for fsgrid.